src/libs/antares/study/study.cpp

```diff
@@ -159,6 +159,7 @@ void Study::createAsNew()
 
     // Simulations
     parameters.reset();
+    parameters.renewableGeneration.rgModelling = Antares::Data::rgClusters;
```

Comment:
Add a comment to explain why we set this value here, possibly

// GUI : use renewable clusters when creating a study